helm/chart-operator/values.yaml

```diff
@@ -1,6 +1,8 @@
 clusterDNSIP: 172.31.0.10
 
 e2e: false
+cnr:
+  address: https://quay.io
```

Comment:
We should remove this from master. Since we removed the chartconfig controller we no longer use cnr.

Reply:
Right, I will remove this thing in master for the next PR.